assets/tests/components/ladderPage.test.tsx

```diff
@@ -557,11 +557,11 @@ describe("LadderPage", () => {
     expect(container.querySelector(".c-route-ladder__header")).toBeVisible()
 
     expect(
-      container.querySelector(".c-route-ladder__dropdown-button")
+      screen.getByRole("button", { name: "1 Route Options" })
```

Comment:
note/nit(non-blocking): btw, I think this `expect().toBeVisible()` is redundant? I'd want to check, but I'm fairly certain you can't use `screen.getBy` to get hidden elements without passing the `{ hidden: true }` option, and it'll stop the test if it can't find it. So both asserts that it's visible and clicks it.
```ts
    await userEvent.click(
      screen.getByRole("button", { name: "1 Route Options" })
    )
```

Reply:
Yeah I was thinking the same thing, but at the time I didn't want to unilaterally get rid of it.

Now it won't be unilateral!